Resources/Prototypes/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```diff
@@ -150,6 +150,18 @@
       gun_magazine: !type:ContainerSlot
       gun_chamber: !type:ContainerSlot
 
+- type: entity
+  name: silencer
+  parent: WeaponPistolCobra
```

Comment:
this will be visible in bags

Reply:
yes but how else can you interact with it in storage?